charts/alluxio/templates/master/statefulset.yaml

```diff
@@ -173,11 +173,25 @@ spec:
           {{- if .Values.master.resources  }}
 {{ include "alluxio.master.resources" . | indent 10 }}
           {{- end }}
+          {{- if eq .Values.master.mountConfigStorage "configmap"}}
+          command: ["/etc/fluid/scripts/entrypoint_replace_secrets.sh"]
```

Comment:
I think we also name the script `/etc/fluid/scripts/entrypoint.sh` .

Reply:
ok, done.